product_tier_validation/data/product_state_data.xml

```diff
@@ -1,7 +1,4 @@
 <odoo noupdate="1">
-    <record id="product_state.product_state_draft" model="product.state">
```

Comment:
this state should stay no?

Reply:
This data already have one in `product_state` module

Reply:
I think we should go further here as I don't understand why we should change the product states codes...